content/jobs.md

```
@@ -97,17 +97,56 @@ Schema:
-    formally validated by the Director when registering a release job.
+    `description`, `default` and `example` are displayed by the [release
+    index](/releases). In turn, other keys like `type` are used only for
+    convenience, like Concourse does `env` keys in the [“web” job
+    definition][concourse_web_spec]. For formal validation by the Director
+    when registering a release job, see [property
+    validation](#property-validation).
 
 [concourse_web_spec]: https://github.com/concourse/concourse-bosh-release/blob/8d2cfa0/jobs/web/spec#L68-L71
 
 ---
+## Property validation {: #property-validation }
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)